README.MD

```diff
@@ -0,0 +1,23 @@
+# Vattenfall bookstore
```

Comment:
# Vattenfall Bookstore

## Application overview
This is an application about books and authors. The code is written in java and Spring Boot 3.x. It uses gradle to manage dependencies.
The app exposes a single endpoint for getting information about books and there is a persistence layer, but the database is not configured 
(you can configure it if you need to, but it is not required to achieve the goal).

## Objective
Unfortunately the application was written by a not very experienced craftsman, so there are no tests. Your goal will be to configure the project and write the tests.
You can choose any test library you prefer (Spock+Groovy, Kotest+Kotlin, jUnit+Java). 
If you have no preference you can choose the Spock framework and write tests in Groovy. 
Remember that the main goal is to write tests and cover all important code branches.
It is up to you to decide what type and how many tests you will write, what tools or libraries you will use and so on. 
The goal is to have a well-tested bug-free project.

## Why
You may ask why such a task? So the reason is that we in the team put a lot of emphasis on testing because we see a lot of value in testing.
So the reason is simple We would like to see if your approach to application testing is similar to ours :-)

## How it
A repository with the project is available on github https://github.com/jgprogram/rekru-proj. 
If you have a github account you can create a fork from the repo and when the job is done create a PullRequest/MergeRequest from your fork to the main repo.
If you don't have a github account you can also clone the project, write the code archive it in a zip and send it to the recruiting maintainer.

Translated with www.DeepL.com/Translator (free version)